health/health.d/backend.conf

```diff
@@ -1,10 +1,11 @@
 # Alert that backend will be disabled soon
    alarm: backend_metrics_eol
       on: netdata.backend_metrics
-   units: bytes
-    calc: sum($sent) + sum($buffered)
+   units: seconds ago
+    calc: $now - $last_collected_t 
    every: 5s
-    warn: $this != 0
+    warn: $this > 0
+    crit: $this > 1800
```

Comment:
```
    warn: $this > 0
    crit: $this > 1800
```

=>

We need to check only if the alarm is up (units - boolean) and level should be `warn`, no need for `crit`.

I didnt think how to make it, but idea is:

 - check if alarm is up (old backend are being used)
 - if it is - warn

Reply:
I tend to agree with you, I added the `crit` to give some emphasys for the fact that nothing was done after half hour.
I will reevaluate this late today.

Reply:
i think change units to `boolean` and make warn condition `warn: $this >= 0` will do

Reply:
yes, it makes sense, I changed it.
Thank you very much!